grafana/base/grafana.yaml

```diff
@@ -0,0 +1,73 @@
+---
+apiVersion: integreatly.org/v1alpha1
+kind: Grafana
+metadata:
+    annotations:
+        argocd.argoproj.io/sync-wave: "1"
+    name: grafana
+spec:
+    baseImage: quay.io/operate-first/grafana:7.1.1
+    config:
+        security:
+            admin_user: "root"
+            admin_password: "secret"
+        users:
+            allow_sign_up: false
+            auto_assign_org: true
+            auto_assign_org_role: Viewer
+            viewers_can_edit: true
+        auth.anonymous:
+            enabled: true
+            org_role: Viewer
+        log:
+            level: warn
+            mode: console
+    dashboardLabelSelector:
+    -   matchExpressions:
+        -   key: app
+            operator: In
+            values:
+            - grafana
+    containers:
+    -   args:
+        - -provider=openshift
+        - -pass-basic-auth=false
+        - -https-address=:9091
+        - -http-address=
+        - -email-domain=*
+        - -upstream=http://localhost:3000
+        - '-openshift-sar={"resource": "namespaces", "verb": "get"}'
+        - -openshift-service-account=grafana-serviceaccount
+        - -openshift-ca=/etc/pki/tls/cert.pem
+        - -openshift-ca=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+        - -tls-cert=/etc/tls/private/tls.crt
+        - -tls-key=/etc/tls/private/tls.key
+        - -cookie-secret=SECRET
+        - -skip-auth-regex=^/metrics,/api/datasources,/api/dashboards
+        - '-openshift-sar={"resource": "namespaces", "verb": "get", "resourceName":
+            "opf-monitoring", "namespace": "opf-monitoring"}'
+        image: registry.redhat.io/openshift4/ose-oauth-proxy:v4.7
+        name: oauth-proxy
+        ports:
+        -   containerPort: 9091
+            name: oauth-proxy
+        resources: {}
+        volumeMounts:
+        -   mountPath: /etc/tls/private
+            name: secret-grafana-tls
+            readOnly: false
+    ingress:
+        enabled: false
+    secrets:
+    - grafana-tls
+    service:
+        annotations:
+            service.alpha.openshift.io/serving-cert-secret-name: grafana-tls
+        ports:
+        -   name: oauth-proxy
+            port: 9091
+            protocol: TCP
+            targetPort: oauth-proxy
+    serviceAccount:
+        annotations:
+            serviceaccounts.openshift.io/oauth-redirectreference.primary: '{"kind":"OAuthRedirectReference","apiVersion":"v1","reference":{"kind":"Route","name":"grafana"}}'
```

Comment:
Please add the route resource as well.

https://github.com/4n4nd/odh/blob/2b74f0a0782f4ef6da22e5a7798121ef17a95295/odh-manifests/grafana/grafana/base/route.yaml

Reply:
Resolved in latest commit.